tools/ktx/command_compare.cpp

```diff
@@ -0,0 +1,2465 @@
+            \- DFD section differences can be ignored with --ignore-dfd all or --ignore-dfd
+               all-except-color-space. <br />
+            \- BDFD bytesPlane value differences can be ignored with --ignore-bdfd-bytesplane. <br />
+            \- SGD section differences can be ignored with --ignore-sgd all or --ignore-sgd
+               payload. <br />
+        </dd>
+        <dt>\--per-pixel-output all | &lt;number&gt; | none</dt>
+        <dd>Controls whether per pixel / texel block difference output is generated when
+            --content is set to @b image:
+            @b all - Every single difference is output (may result in a very large output). <br />
+            @b &lt;number&gt; - At most the specified number of differences are output. <br />
+            @b none - No per pixel / texel block differences are output. <br />
+            The default mode is @b none to limit the verbosity of the output.
+        <dt>\--allow-invalid-input</dt>
+        <dd>Perform best effort comparison even if any of the input files are invalid.</dd>
+        <dt>\--ignore-format-header<dt>
+        <dd>Ignore the vkFormat and typeSize fields in the file header. <br />
+            Note: useful when comparing textures with and without BasisLZ encoding, respectively.</dd>
+        <dt>\--ignore-supercomp</dt>
+        <dd>Ignore supercompression scheme in the file header. <br />
+            Note: use the --ignore-sgd option to also ignore the SGD section, if needed.</dd>
+        <dt>\--ignore-index all | level | none</dt>
+        <dd>Controls the comparison of index entries in the file headers. Possible options are: <br />
+            @b all - Ignore all index entries. <br />
+            @b level - Ignore level index entries only. <br />
+            @b none - Do not ignore any index entries. <br />
+            The default mode is @b none, meaning that all index entries will be compared.
+        </dd>
+        <dt>\--ignore-dfd all | all-except-color-space | unknown | extended | none</dt>
+        <dd>Controls the comparison of DFD blocks. Possible options are: <br />
+            @b all - Ignore all DFD blocks. <br />
+            @b all-except-color-space: Ignore all DFD blocks except the color space information
+                in the basic DFD block. <br />
+            @b unknown - Ignore any unrecognized DFD blocks. <br />
+            @b extended - Ignore all DFD blocks except the basic DFD block. <br />
+            @b none - Do not ignore any DFD blocks. <br />
+            The default mode is @b none, meaning that all DFD entries will be compared.
+        </dd>
+        <dt>\--ignore-bdfd-bytesplane</dt>
+        <dd>Ignore BDFD bytesPlane values. Useful when comparing supercompressed files with
+            non-supercompressed ones, as bytesPlane is set to zero for supercompressed files.</dd>
+        <dt>\--ignore-metadata all | &lt;keys&gt; | none</dt>
```

Comment:
all | key<,...> | none

Reply:
The format `key<,...>` does not make sense, `<x>` means in CLI option format that it's a named value rather than a literal, `[x]` is for optional arguments, at most it should be something like `<key>[,<key>...]`, but that would be unnecessarily verbose IMO, as the description clearly indicates it is supposed to be a comma separated list.

Reply:
Somehow a reply I typed earlier today has gone missing. I confused the meanings of `<>` and `[]`. I like it to be clear from a quick scan of the option without having to read the descriptions. How about `<key>[,...]` to reduce the verbosity. I sure I've seen this kind of abbreviation used elsewhere for repeatable arguments.

Reply:
Sure, fine by me. I still think it's unconventional, but certainly better.

Reply:
Updated.